upgrades/backend.go

```diff
@@ -16,6 +15,7 @@ import (
 	"github.com/juju/juju/environs/config"
 	"github.com/juju/juju/state"
 	raftleasestore "github.com/juju/juju/state/raftlease"
+	"github.com/juju/replicaset/v2"
```

Comment:
this import is in the wrong stanza, belongs with juju/errors.

Reply:
@hmlanigan what did you mean by this? :point_up:

Reply:
imports for juju are in 3 stanzas/groups, std go libs, non juju/juju libs and juju/juju libs. "github.com/juju/replicaset/v2" belongs in the 2nd.

Reply:
Ah ok I didn't pay attention to the `juju/juju/` vs `juju/`, I'll fix it :+1: good catch!